### PR TITLE
fix integration test failing due to zero division

### PR DESCRIPTION
**Pull request by n1ru4l**
### Background

There seems to be an edge case where, the materialised views are not in sync when a operation is executed, which results in  a zero devision causing a runtime exception in the GraphQL layer.

This often fails integration tests. In a real world scenario this does not seems to happen as there is much more traffic and a conditional breaking change is not checked 1 second after the first request for was done... No error was ever reported for this to sentry.

This is super annoying and occurs quite frequently. I am open for better solutions here...

### CODE REVIEW

File: packages/services/api/src/modules/schema/providers/breaking-schema-changes-helper.ts [hunk 1]
```diff
@@ -36,15 +36,31 @@ export class BreakingSchemaChangeUsageHelper {
 
     return {
       topAffectedOperations: schemaChange.usageStatistics.topAffectedOperations.map(operation => {
-        const percentage = (operation.count / metadata.usage.totalRequestCount) * 100;
+        // Note:
+        // "metadata.usage.totalRequestCount" is sometimes 0 in case of a integration test
+        // causing a zero devision and GraphQL exception,
+        // because we aggressively poll for the schema change after publishing usage data
+        // it seems like clickhouse slighty lags behind for the materialized view here.
+        // since it only happens in contetx of an integration test (no production issues)
```
Comment (n1ru4l) [inline packages/services/api/src/modules/schema/providers/breaking-schema-changes-helper.ts]:
```suggestion
        // since it only happens in context of an integration test (no production issues)
```

File: packages/services/api/src/modules/schema/providers/breaking-schema-changes-helper.ts [hunk 2]
```diff
@@ -36,15 +36,31 @@ export class BreakingSchemaChangeUsageHelper {
...
         };
       }),
       topAffectedClients: schemaChange.usageStatistics.topAffectedClients.map(client => {
-        const percentage = (client.count / metadata.usage.totalRequestCount) * 100;
+        // Note:
+        // "metadata.usage.totalRequestCount" is sometimes 0 in case of a integration test
+        // causing a zero devision and GraphQL exception,
+        // because we aggressively poll for the schema change after publishing usage data
+        // it seems like clickhouse slighty lags behind for the materialized view here.
+        // since it only happens in contetx of an integration test (no production issues)
```
Comment (n1ru4l) [inline packages/services/api/src/modules/schema/providers/breaking-schema-changes-helper.ts]:
```suggestion
        // since it only happens in context of an integration test (no production issues)
```
